pkg/mermaid/mermaidsequencediagram.go

```diff
@@ -0,0 +1,157 @@
+package mermaid
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+
+	"github.com/anz-bank/sysl/pkg/sysl"
+)
+
+type sequencepair struct {
+	firstapp, secondapp, endpoint string
+}
+
+const projDir = "../../"
+
+var startElse = regexp.MustCompile("^else.*")
+var isElse = regexp.MustCompile("^else$")
+var isLoop = regexp.MustCompile("^loop.*")
+
+func GenerateMermaidSequenceDiagram(m *sysl.Module, appname string, epname string) (string, error) {
+	var sp []sequencepair
+	return generateMermaidSequenceDiagramHelper(m, appname, epname, "...", 1, &sp, true)
```

Comment:
`return generateMermaidSequenceDiagramHelper(m, appname, epname, "...", 1, &[]sequencepair{}, true)`

Reply:
Got it!